modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -619,6 +619,15 @@
     <string name="profile_amount_free">%s free</string>
     <string name="profile_auto_download_settings">Auto download settings</string>
     <string name="profile_cancel_subscription">Cancel subscription</string>
+    <string name="profile_cancel_confirm_subtitle">This will change your plan to a free account.</string>
+    <string name="profile_cancel_confirm_sub_expiry">Your current subscription will remain active until %1$s.</string>
+    <string name="profile_cancel_confirm_sub_expiry_date_fallback">your expiration date</string>
+    <string name="profile_cancel_confirm_item_plus">Access to Pocket&#160;Casts Plus features will be locked after this date.</string>
+    <string name="profile_cancel_confirm_item_folders">Your folders will be removed and their contents will move back to the Podcasts screen.</string>
+    <string name="profile_cancel_confirm_item_uploads">All files uploaded to your Pocket&#160;Casts account will be deleted (but downloaded files on your mobile devices will remain)</string>
```

Comment:
Nitpick: this is the only item that does not end with a period. Was that intentional?

Reply:
It was an oversight. Added in a7d9371.